Common/Serialize/SerializeMap.h

```diff
@@ -62,9 +62,9 @@ void DoMap(PointerWrap &p, M &x, typename M::mapped_type &default_val) {
 template<class K, class T>
 void Do(PointerWrap &p, std::map<K, T *> &x) {
 	if (p.mode == PointerWrap::MODE_READ) {
-		for (auto it = x.begin(), end = x.end(); it != end; ++it) {
-			if (it->second != nullptr)
-				delete it->second;
+		for (const auto &[_,value] : x) {
```

Comment:
Wow, this is getting a little to modern for me haha.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18559/commits/37ce2a4de179e6931b8f7e5de4c78728ac0c5c5b